pkg/apis/admissionregistration/types.go

```diff
@@ -154,9 +154,8 @@ type ValidatingAdmissionPolicySpec struct {
 	// Required.
 	MatchConstraints *MatchResources
 
-	// Validations contain CEL expressions which is used to apply the validation.
-	// A minimum of one validation is required for a policy definition.
-	// Required.
+	// Validations contain CEL expressions which are used to validate admission requests.
+	// +optional
```

Comment:
We discussed in the last API review. This allows developers to avoid putting a no-op (`true`) `Validations` when all they want to do is use `AuditAnnotations`.

Good call on the documentation. I've added that.